driver/kubernetes/kubernetes.go

```diff
@@ -76,6 +78,7 @@ func (k *Driver) Handles(imagetype string) bool {
 // Config returns the Kubernetes driver configuration options.
 func (k *Driver) Config() map[string]string {
 	return map[string]string{
+		"IN_CLUSTER":      "Connect to the ambient cluster",
```

Comment:
nit: what is an "ambient" cluster?

Reply:
Oops, that's not what I meant to write. It should be "Connect to the cluster using in-cluster environment variables"